ecommerce/.env.dist

```diff
@@ -1,5 +0,0 @@
-DEBUG=True
```

Comment:
bu dosyayi neden sildik?

Reply:
Önceki hafta db fetch&mergede hata yaşadığım için dbyi komple silip yeniden create ettim. O esnada kayboldu sanırım